examples/extended_validation/src/tests.rs

```diff
@@ -0,0 +1,89 @@
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::{ContentType, Status};
+
+use super::rocket;
+
+fn test_login(username: &str,
```

Comment:
I dislike the formatting here. Maybe we can use shorter variable names like `user` and `pass` to try to get this to fit?